rules/aws/credential_access_aws_iam_assume_role_brute_force.toml

```diff
@@ -47,6 +47,6 @@ name = "Credential Access"
 reference = "https://attack.mitre.org/tactics/TA0006/"
 
 [rule.threshold]
-field = [""]
+field = []
```

Comment:
i thought there were two affected rules?

Reply:
Just checked again and only see this one